src/main/java/MyThread.java

```diff
@@ -0,0 +1,17 @@
+import org.apache.log4j.Logger;
+
+public class MyThread extends Thread {
+    private static final Logger logger = Logger.getLogger(MyThread.class);
+    private Counter counter;
+
+    @Override
+    public void run() {
+        counter = new Counter();
```

Comment:
you create two different counters. That's not race :( You need created one in main methods and then increase the count by two threads in the same time.

Reply:
done